### PR TITLE
PR 1749 tweaks

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/feldon_ronom_excavator.txt [hunk 1]
```diff
@@ -4,12 +4,13 @@ Types:Legendary Creature Human Artificer
...
 SVar:TrigExile:DB$ Dig | Defined$ You | DigNum$ X | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBChoose
-SVar:DBChoose:DB$ ChooseCard | Choices$ Card.IsRemembered | ChoiceZone$ Exile | Defined$ You | Amount$ 1 | Mandatory$ True | SubAbility$ DBEffect
-SVar:DBEffect:DB$ Effect | RememberObjects$ ChosenCard | StaticAbilities$ Play | Duration$ UntilTheEndOfYourNextTurn | ForgetOnMoved$ Exile | SubAbility$ DBCleanup
-SVar:Play:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may play the exiled card this turn.
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBChoose:DB$ ChooseCard | Choices$ Card.IsRemembered | ChoiceZone$ Exile | Mandatory$ True | SubAbility$ DBCleanup1
+SVar:DBCleanup1:DB$ Cleanup | ClearRemembered$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ Play | Duration$ UntilTheEndOfYourNextTurn | ForgetOnMoved$ Exile | SubAbility$ DBCleanup2
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/feldon_ronom_excavator.txt:11]: ForgetOnMoved doesn't work if nothing is Remembered

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/feldon_ronom_excavator.txt]: ah good eye

File: forge-gui/res/cardsfolder/upcoming/feldon_ronom_excavator.txt [hunk 2]
```diff
@@ -4,12 +4,12 @@ Types:Legendary Creature Human Artificer
 PT:2/2
 K:Haste
 K:CARDNAME can't block.
-T:Mode$ DamageDoneOnce | ValidTarget$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigExile |  TriggerDescription$ Whenever NICKNAME is dealt damage, exile that many cards from the top of your library. Choose one of them. Until the end of your next turn, you may play that card.
+T:Mode$ DamageDoneOnce | ValidTarget$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever NICKNAME is dealt damage, exile that many cards from the top of your library. Choose one of them. Until the end of your next turn, you may play that card.
 SVar:TrigExile:DB$ Dig | Defined$ You | DigNum$ X | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBChoose
-SVar:DBChoose:DB$ ChooseCard | Choices$ Card.IsRemembered | ChoiceZone$ Exile | Defined$ You | Amount$ 1 | Mandatory$ True | SubAbility$ DBEffect
+SVar:DBChoose:DB$ ChooseCard | Choices$ Card.IsRemembered | ChoiceZone$ Exile | Mandatory$ True | ForgetOtherRemembered$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/feldon_ronom_excavator.txt:9]: Missing `SubAbility$ DBEffect`

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/feldon_ronom_excavator.txt:9]: 🤦‍♂️

**Review by tool4ever — APPROVED** [review]

**Pull request merged**